src/common/libjob/jobspec1.c

```diff
@@ -78,7 +80,9 @@ int flux_jobspec1_attr_pack (flux_jobspec1_t *jobspec,
     return rc;
 }
 
-int flux_jobspec1_attr_check (flux_jobspec1_t *jobspec, char *errbuf, int errbufsz)
+int flux_jobspec1_attr_check (flux_jobspec1_t *jobspec,
+                              char *errbuf,
+                              int errbufsz)
```

Comment:
I prefer the `*_error_t` style of error message passing over the `char *errbuf, int errbufsz` style. Maybe over time we could semi-standardize on that interface? IMO, removing the extra parameter makes it easy on the caller (among other benefits).

I realize this is just a style preference though.

Reply:
I prefer that too.  OK, I'll see if I can make that change.

Reply:
OK, just pushed a commit that makes that change.